### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by LaskoA**
Please check it, thank you.

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command):
+    with open(command[3:12], 'r') as first, open(command[12:], 'a') as second:
```
Comment (IvanRamyk) [inline app/main.py:2]: Choose more descriptive names for files, e.g: `source` or `destination`

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

**Comment by LaskoA** [comment]
I chose "old" and "new" this time - will it work? :)

File: app/main.py [hunk 2]
```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command):
+    with open(command[3:12], 'r') as old, open(command[12:], 'a') as new:
```
Comment (IvanRamyk) [inline app/main.py:2]: Will slices work with filenames of different lengths? 
I suppose, won't. Use `split` to get filenames

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

**Comment by LaskoA** [comment]
Please check now with new logic (command = command[3:].split(" "))

**Pull request closed without merging**